src/components/views/dialogs/AddExistingToSpaceDialog.tsx

```diff
@@ -266,6 +266,8 @@ export const AddExistingToSpace: React.FC<IAddExistingToSpaceProps> = ({
 
     // only count spaces when alone as they're shown on a separate modal all on their own
     const numSpaces = (spacesRenderer && !dmsRenderer && !roomsRenderer) ? spaces.length : 0;
+    const numRooms = roomsRenderer ? rooms.length : 0;
+    const numDms = dmsRenderer ? dms.length : 0;
 
     let noResults = true;
     if ((roomsRenderer && rooms.length > 0) || (dmsRenderer && dms.length > 0) || (numSpaces > 0)) {
```

Comment:
should this be using the same thing too?

Reply:
Might as well to simplify the expression